pkg/controller/util.go

```diff
@@ -24,45 +22,13 @@ const DataVolName = "cdi-data-vol"
 const ImagePathName = "image-path"
 const socketPathName = "socket-path"
 
-// return a pvc pointer based on the passed-in work queue key.
-func (c *ImportController) pvcFromKey(key interface{}) (*v1.PersistentVolumeClaim, bool, error) {
-	obj, exists, err := c.objFromKey(c.pvcInformer, key)
-	if err != nil {
-		return nil, false, errors.Wrap(err, "could not get pvc object from key")
-	} else if !exists {
-		return nil, false, nil
-	}
-
-	pvc, ok := obj.(*v1.PersistentVolumeClaim)
-	if !ok {
-		return nil, false, errors.New("Object not of type *v1.PersistentVolumeClaim")
-	}
-	return pvc, true, nil
-}
-
-func (c *ImportController) objFromKey(informer cache.SharedIndexInformer, key interface{}) (interface{}, bool, error) {
-	keyString, ok := key.(string)
-	if !ok {
-		return nil, false, errors.New("keys is not of type string")
-	}
-	obj, ok, err := informer.GetIndexer().GetByKey(keyString)
-	if err != nil {
-		return nil, false, errors.Wrap(err, "error getting interface obj from store")
-	}
-	if !ok {
-		return nil, false, nil
-	}
-	return obj, true, nil
-}
-
-func checkPVC(pvc *v1.PersistentVolumeClaim) bool {
+func checkPVC(pvc *v1.PersistentVolumeClaim, annotation string) bool {
 	if pvc.DeletionTimestamp != nil {
 		return false
 	}
-
-	// check if we have proper AnnEndPoint annotation
-	if !metav1.HasAnnotation(pvc.ObjectMeta, AnnEndpoint) {
-		glog.V(2).Infof("pvc annotation %q not found, skipping pvc \"%s/%s\"\n", AnnEndpoint, pvc.Namespace, pvc.Name)
+	// check if we have proper annotation
+	if !metav1.HasAnnotation(pvc.ObjectMeta, annotation) {
+		glog.V(3).Infof("pvc annotation %q not found, skipping pvc\n", annotation)
```

Comment:
Can you have this look like:
```go
glog.V(2).Infof("pvc annotation %q not found, skipping pvc \"%s/%s\"\n", AnnEndpoint, pvc.Namespace, pvc.Name)
```
Some of the e2e tests look for that string, this is causing the failure in the CI